### DISCUSSION TITLE
The star rating in the beatmap details box doesn't update correctly after changing "Wind Up / Wind Down" mod's final rate

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#15326] [Q&A] [answered]
For some reason, the star rating in the "Details" tab of the carousel doesn't update after changing the final rate of the `Wind Up` / `Wind Down` mod.
As you can see from the following video, the `Difficulty Adjust` mod updates the values correctly, but with the `Wind Up` / `Wind Down` mod, you can notice the star rating updating in the wedge but not in the "Details" tab (unless you re-enter the carousel):

https://user-images.githubusercontent.com/43073074/139157349-3d6313f4-1b32-45d2-a401-857aa38bc035.mp4

**Comment by Gabixel** [comment]
This also happens with `Double Time` / `Half Time` and their variants... so the problem is more about mods that change speed in general.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15329
